### DISCUSSION TITLE
Welcome to safe-core-sdk Discussions!

### DISCUSSION THREAD

**Discussion by germartinez** [safe-global/safe-core-sdk#102] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UmangGitHub** [comment]
Hello @germartinez,
Hope you are all well.

I am building a back-end in Node..js which will be integrating the Gnosis Safe functionalities. I am failing to run the following code:

```
import SafeServiceClient from '@gnosis.pm/safe-service-client'

const transactionServiceUrl = 'https://safe-transaction.gnosis.io'
const safeService = new SafeServiceClient(transactionServiceUrl)
```

However, I am getting the following error:
> TypeError: SafeServiceClient is not a constructor

Really looking forward to resolve this problem that I am facing.

Thank you.

**Reply by germartinez** [reply]
Hi, the code is correct. I think it is an issue with not supporting ES6 imports

**Reply by UmangGitHub** [reply]
![Error - safeServiceClient](https://user-images.githubusercontent.com/40272704/156142347-be46fb72-f5bf-49e8-bc10-307237658682.png)

**Reply by UmangGitHub** [reply]
I have used _require_ this time, still facing the same problem.

I believe there is something very simple that I am doing wrong. Would really appreciate to get sorted.

Looking forward.

**Reply by CredittoGH** [reply]
what do I need to do this was the first attempt that nothing went wrong

**Comment by CredittoGH** [comment]
What do I need to do?
